ntc_templates/templates/aruba_aoscx_show_vsf_detail.textfsm

```diff
@@ -1,18 +1,24 @@
 Value MEMBER_ID (\S+)
+Value TOPOLOGY (\S+)
+Value NAME (\S+)
 Value MAC_ADDRESS (\S+)
-Value STATUS (.*)
+Value STATUS (\S+.*)
+Value SERIAL_NUMBER (\S+)
 Value CPU (\d+%)
 Value MEMORY (\d+%)
 Value VSF_LINK_1 (.*)
 Value VSF_LINK_2 (.*)
 
 Start
-  ^VSF\s*${MEMBER_ID}
+  ^VSF\s+${MEMBER_ID}
+  ^\s+Topology\s+:\s+${TOPOLOGY}
+  ^\s+Name\s+:\s+${NAME}
   ^Member.* -> Continue.Record
   ^Member\s+ID\s+:\s+${MEMBER_ID}
-  ^\s+MAC\s+Address\s+:\s+${MAC_ADDRESS}                                                                          
-  ^\s+Status\s+:\s+${STATUS}                                                                       
-  ^\s+CPU\sUtilization\s+:\s+${CPU}                                                                      
+  ^\s+MAC\s+Address\s+:\s+${MAC_ADDRESS}
+  ^\s+Status\s+:\s+${STATUS}
+  ^\s+Serial\s+Number\s+:\s+${SERIAL_NUMBER}
+  ^\s+CPU\s+Utilization\s+:\s+${CPU}
   ^\s+Memory\sUtilization\s+:\s+${MEMORY}
```

Comment:
I was only looking at the changed lines earlier and not those that were unchanged. Oops.

We could change the whitespace regex for Memory Utilization to be future-proofed to accept one or more spaces instead of just one. Not only does this future-proof, but also follows the pattern of other rules using `\s+`.

```suggestion
  ^\s+Memory\s+Utilization\s+:\s+${MEMORY}
```

Reply:
Ya fair, I was curious why some items had \s+ and others not.